Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -0,0 +1,438 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
That wouldn't be an issue at all my changes were very basic anyway.
Up to you !